support-frontend/assets/helpers/dateConversions.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+const formatMachineDate = (date: Date) =>
```

Comment:
👍 good to move these to their own module

Reply:
thanks I'm glad it turned out OK!